src/plugins/GUITestBase/src/tests/common_scenarios/Assembling/extract_consensus/GTTestsAssemblyExtractConsensus.cpp

```diff
@@ -0,0 +1,327 @@
+/**
+ * UGENE - Integrated Bioinformatics Tools.
+ * Copyright (C) 2008-2020 UniPro <ugene@unipro.ru>
+ * http://ugene.net
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+ * MA 02110-1301, USA.
+ */
+
+#include "GTTestsAssemblyExtractConsensus.h"
+
+#include <QApplication>
+#include <QFileInfo>
+
+#include "GTUtilsDashboard.h"
+#include "GTUtilsLog.h"
+#include "GTUtilsTaskTreeView.h"
+#include "GTUtilsWizard.h"
+#include "GTUtilsWorkflowDesigner.h"
+#include "base_dialogs/DefaultDialogFiller.h"
+#include "base_dialogs/GTFileDialog.h"
+#include "base_dialogs/MessageBoxFiller.h"
+#include "drivers/GTKeyboardDriver.h"
+#include "primitives/GTComboBox.h"
+#include "primitives/GTLineEdit.h"
+#include "primitives/GTMenu.h"
+#include "primitives/GTToolbar.h"
+#include "primitives/GTWidget.h"
+#include "runnables/ugene/plugins/workflow_designer/WizardFiller.h"
+#include "utils/GTUtilsDialog.h"
+
+namespace U2 {
+
+namespace GUITest_assembly_extract_consensus {
+
+struct ExtractConsensusWizardScenario : CustomScenario {
```

Comment:
struct extends class: avoid this mix unless you have a reason for that.

Reply:
At the beginning, I thought that I would need to change the fields of the class/structure, in this regard, I did not find a reason to make unnecessary getters and setters, so I needed a structure with its public fields. However, later I realized that I could not reuse the same created object, because it is deleted, and therefore the need for the structure is gone. I'll fix that. Thank you for your comment.